examples/at_circular_piston_3D/at_circular_piston_3D.py

```diff
@@ -0,0 +1,244 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from copy import deepcopy
+
+from kwave.data import Vector
+
+from kwave.kgrid import kWaveGrid
+from kwave.kmedium import kWaveMedium
+from kwave.ksensor import kSensor
+from kwave.ksource import kSource
+from kwave.utils.filters import extract_amp_phase
+from kwave.utils.math import round_even
+from kwave.utils.kwave_array import kWaveArray
+from kwave.utils.signals import create_cw_signals
+
+from kwave.kspaceFirstOrder3D import kspaceFirstOrder3D
+
+from kwave.options import SimulationOptions, SimulationExecutionOptions
+
+def L2_error(x: np.ndarray, y: np.ndarray, ord=None) -> float:	

```

Comment:
I believe this could be replaced with [`np.linalg.norm` ](https://numpy.org/doc/stable/reference/generated/numpy.linalg.norm.html).